File: lev1/없는 숫자 더하기.py
def solution(numbers):
    ans = 0
    for n in range(10):
        if n not in numbers:
            ans += n
    return ans


print(solution([1,2,3,4,6,7,8,0]))
print(solution([5,8,4,0,6,7,9]))
print(solution([0]))
print(solution([0,1,2,3,4,5,6,7,8]))
print(solution([1,2,3,4,5,6,7,8,9]))

File: lev1/없는숫자더하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// numbers_len은 배열 numbers의 길이입니다.
int solution(int numbers[], size_t numbers_len) {
    int total = 0;
    for(int i=0; i<numbers_len; i++)
        total += numbers[i];
    return 45-total;
}


int main(){
    int arr1[] = {1,2,3,4,6,7,8,0};
    printf("%d\n", solution(arr1, 8));//14
    int arr2[] = {5,8,4,0,6,7,9};
    printf("%d\n", solution(arr2, 7));//6
    return 0;
}
